container-core/src/main/java/com/yahoo/restapi/RestApiImpl.java

```diff
@@ -151,10 +151,11 @@ private static Route createDefaultRoute() {
 
     private static List<ExceptionMapperHolder<?>> combineWithDefaultExceptionMappers(
             List<ExceptionMapperHolder<?>> configuredExceptionMappers, boolean disableDefaultMappers) {
-        List<ExceptionMapperHolder<?>> exceptionMappers = new ArrayList<>(configuredExceptionMappers);
+        List<ExceptionMapperHolder<?>> exceptionMappers = new ArrayList<>();
```

Comment:
This will no longer make is possible to override a single builtin handlers (as they will always have priority).